client/pages/DataExport/DataExport.tsx

```diff
@@ -1,30 +1,40 @@
-import './DataExport.scss'
-import React from 'react'
+import React, { useEffect } from 'react'
 
-// import { Objects } from '@core/utils'
+import { useCountryIso } from '@client/hooks'
+import { useAppDispatch } from '@client/store'
 // import { useDataExportCountries, useDataExportSelection } from '@webapp/store/page/dataExport'
+import { DataExportActions, useDataExportCountries } from '@client/store/pages/dataExport'
+import { Objects } from '@core/utils'
 
-// import CountrySelect from './CountrySelect'
+import CountrySelect from './CountrySelect'
+import './DataExport.scss'
 // import VariableSelect from './VariableSelect'
 // import ColumnSelect from './ColumnSelect'
 // import ResultsTable from './ResultsTable'
 
 const DataExport: React.FC = () => {
+  const dispatch = useAppDispatch()
+
+  const countryIso = useCountryIso()
   // const assessmentSection = useParamSection()
-  // const countries = useDataExportCountries()
+  const countries = useDataExportCountries()
   // const selection = useDataExportSelection(assessmentSection)
 
   // const hasSelection =
   //   !Objects.isEmpty(selection.countryISOs) &&
   //   !Objects.isEmpty(selection.sections[assessmentSection].columns) &&
   //   !Objects.isEmpty(selection.sections[assessmentSection].variables)
 
-  // if (Objects.isEmpty(countries)) return null
+  useEffect(() => {
+    dispatch(DataExportActions.reset())
```

Comment:
It should be in the return function as agreed verbally . Have you tried ?

Reply:
done